### PR TITLE
Doctrine migrations 3.0 post 

### PR DESCRIPTION
**Pull request by goetas**
Not sure it makes sense this pull request, but this re-publishes on the official doctrine blog the [article](https://www.goetas.com/blog/released-doctrinemigrations-30-alpha/) I wrote for the doctrine/migrations 3.0-alpha release.

I have mentioned the original post at the bottom of the article and added the canonical url link to avoid google penalties for duplicate content.

If you think is not a good idea, feel free to close the pull request.

### CODE REVIEW

**Comment by malarzm** [comment]
I think it's the right place for such a blog post :) Quickly looking at the files I'd say that there's no point in adding doctrine logo, there should be something already in the repository :)

**Comment by goetas** [comment]
>Quickly looking at the files I'd say that there's no point in adding doctrine logo, there should be something already in the repository :) 

Logo removed! was just a copy/paste mistake :)

**Comment by goetas** [comment]
I'm not familiar with this project, do I have to edit the `BlogPostHydrator` and `BlogPost` classes adding the "canonical" property?

File: templates/layouts/layout.html.twig [hunk 1]
```diff
@@ -26,6 +26,10 @@
             <meta name="robots" content="noindex">
         {% endif %}
 
+        {% if page.canonical %}
```
Comment (SenseException) [inline templates/layouts/layout.html.twig:29]:
```suggestion
        {% if page.canonical is defined and page.canonical %}
```

I did some tests and this change was enough to make the canonical appear when is was provided in the blog post.

Comment (goetas) [reply templates/layouts/layout.html.twig]: fixed

**Comment by SenseException** [comment]
The build is failing because of website configurations in other branches. Probably just #324. The blog post will appear only with a working build. After the fixes there are done and the build works, I'll test, approve + merge the blog post.

**Comment by goetas** [comment]
Thanks

**Comment by SenseException** [comment]
Thank you

**Pull request merged**